source/scpd/tests/LayoutTest.d

```diff
@@ -24,7 +24,7 @@ import std.string;
 import std.traits;
 
 /// Contains the size and offset of a field within a struct
-extern(C++) struct FieldInfo
+extern(C++, class) struct FieldInfo
```

Comment:
https://github.com/bpfkorea/agora/blob/d8154e24ed91844e7730ed646b5c3913cb5ec8fc/source/scpp/extra/DLayoutChecks.cpp#L31
???

Reply:
This is a struct, please revert this change..

Reply:
It's my mistake.
I don't know why the changes keep going back. I don't think I'm familiar with Git.

Reply:
Which commands did you run in git?

Reply:
I used git rebase frequently.
Maybe I used the wrong rebase in the middle